tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -572,3 +574,25 @@ procSuite "WakuNode - RLN relay":
 
     await node1.stop()
     await node2.stop()
+
+  asyncTest "test rln-relay-message-limit":
+    let
+      nodeKey = generateSecp256k1Key()
+      node = newTestWakuNode(nodeKey, parseIpAddress("0.0.0.0"), Port(0))
+
+    await node.mountRelay(@[DefaultPubsubTopic])
+
+    # mount rlnrelay in off-chain mode
+    let wakuRlnConfig = WakuRlnConfig(
+      rlnRelayDynamic: false,
```

Comment:
notice that it is not dynamic, aka not onchain, hence we cannot get the user message limit from the contract. I would suggest to use a different mechanism for the static group manager